schema/rdl_schema_0.1.json

```
@@ -17,16 +16,19 @@
 	],
 	"properties": {
 		"title": {
+			"title": "Dataset title",
 			"type": "string",
-			"description": "Title of the dataset"
+			"description": "The title of the dataset."
 		},
-		"abstract": {
+		"description": {
+			"title": "Description",
 			"type": "string",
-			"description": "Short description of the resource"
+			"description": "A short description of the dataset."
 		},
-		"component": {
+		"risk_data_type": {
+			"title": "Risk data type",
 			"type": "string",
-			"description": "Univocal schema for this contribution (H, E, V or L)",
+			"description": "The datasets risk data type.",
```

Need to add a CSV file for this codelist and add the codelist keywords.

```suggestion
			"description": "The type of risk data included in the dataset, from the closed risk data types codelist.",
```